src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -563,6 +563,20 @@ object TezosDatabaseOperations extends LazyLogging {
       .as[AccountsHistoryRow]
       .map(_.toList)
 
+  /**
+    * Gets inactive bakers from accounts
+    * @param activeBakers needed to filter out them from all of the bakers
+    * @return inactive baker accounts
+    * */
+  def getInactiveBakersFromAccounts(
+      activeBakers: List[String]
```

Comment:
Could we wrap these `strings` as a typed id, or add docs explanation of what should we're expected to pass-in here?

Reply:
I've updated the docs comment with clearer explanation